setup.py

```diff
@@ -26,6 +27,29 @@
 
 os.chdir(root_path)
 
+class Tester(Command):
+    """Cmdclass for `python setup.py test`.
+    Args:
+        Command (distutils.cmd.Command):
+            Abstract base class for defining command classes.
+    """
+
+    description = 'test the code using pytest'
+    user_options: List[Tuple[str, str, str]] = []
+
+    def initialize_options(self):
+        """Set default values for options that this command supports."""
+        pass
+
+    def finalize_options(self):
+        """Set final values for options that this command supports."""
+        pass
+
+    def run(self):
+        """Run pytest."""
+        errno = os.system('python3 -m pytest -v tests/')
```

Comment:
Can you replace this with `subprocess.check_call`?

Reply:
Done.